demos/TicTacToe/Classes/TicTacToeScene.cpp

```diff
@@ -0,0 +1,81 @@
+#include "TicTacToeScene.h"
+
+#include <array>
+#include <cstdlib>
+#include <iostream>
+
+#include "cocos2d.h"
+
+USING_NS_CC;
+
+const int kNumberOfTiles = 9;
+const int kMaxMovesPerPlayer = 1 + kNumberOfTiles / 2;
+const int kTilesX = std::sqrt(kNumberOfTiles);
+const int kTilesY = std::sqrt(kNumberOfTiles);
+const double kScreenWidth = 600;
+const double kScreenHeight = 600;
+const double kTileWidth = (kScreenWidth / kTilesX);
+const double kTileHeight = (kScreenHeight / kTilesY);
+const int kNumberOfPlayers = 2;
+std::array<std::string, kNumberOfPlayers> kMoveImageFileNames = {
```

Comment:
We can't have std::strings as global variables according to the C++ style guide. 

https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables

strings do dynamic allocation and then delete on destruction, which is against the style guide rules. 

Instead either use const char* here, or make it a string* and initialize the value at startup.